test/LowerColorado_TX/test_AnA.yaml

```diff
@@ -38,8 +38,10 @@ compute_parameters:
         wrf_hydro_waterbody_ID_crosswalk_file    : domain/LAKEPARM.nc
         wrf_hydro_waterbody_crosswalk_filter_file: domain/RouteLink.nc
     hybrid_parameters:
-        run_hybrid_routing: False
+        run_hybrid_routing: True

```

Comment:
I'd prefer if we didn't change the parameter values in the testing yaml file. The original yaml file for the Lower Colorado test case is initially set up to run a purely MC simulation with streamflow nudging and reservoir DA.